core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -401,14 +406,15 @@ private Integer nextSchemaIdCounterBatch() throws SchemaRegistryStoreException {
 
     while (true) {
 
-      if (!zkClient.exists(ZOOKEEPER_SCHEMA_ID_COUNTER)) {
+      if (!zkUtils.zkClient().exists(ZOOKEEPER_SCHEMA_ID_COUNTER)) {
```

Comment:
I'm still not 100% on when it is good practice to use parentheses in scala calls without parameters. I think in this case you shouldn't have it.